deploy/jN/start.js

```diff
@@ -189,6 +189,7 @@ require["currentDir"] = new ActiveXObject("Scripting.FileSystemObject").GetFile(
 require["currentModule"] = {exports:{}};	
 
 require("lib/ECMA262.js");
+require("lib/MsgBoxConstants.js");
```

Comment:
Is there a need to import the constants here at the start of the plugin or can it be defered in time to the first access?